ui/src/common/components/PublicNotesList.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { List } from 'immutable';
 
 import InlineList from './InlineList';
+import { SEPARATOR_MIDDLEDOT } from './InlineList/constants';
```

Comment:
those imports are not nice, let's change the `index.js` of `InlineList` to export stuff directly.

```js

export default InlineList;

export { default as InlineUL } from './InlineUL'
export { SEPARATOR_AND, ... } from './constants'
```